secrets/onepassword/backend.go

```diff
@@ -33,15 +32,13 @@ func (b *Backend) Init(parameters map[string]string) error {
 	if err != nil {
 		return fmt.Errorf("Error reading 1password backend parameters: %v", err)
 	}
-
 	b.Vault = parameters["vault"]
 
 	err = b.Client.SignIn(parameters["domain"], parameters["email"], parameters["secretKey"], parameters["masterPassword"])
 	if err != nil {
-		return err
+		fmt.Println(fmt.Sprintf("could not sign in to 1password %s", err))
 	}
-
-	fmt.Println("Signed into 1password successfully.")
+	fmt.Println(fmt.Sprintf("Signed into 1password successfully"))
```

Comment:
same as above, but with `log`

Reply:
Let's fix logging as part of https://github.com/ContainerSolutions/externalsecret-operator/issues/44